lib/Importer/Caffe2.cpp

```diff
@@ -328,6 +328,18 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return;
   }
 
+  if (typeName == "ReplaceNaN") {
+    // Load the input and NaN replacement value:
+    auto input = getNodeValueOrCreateVariableByName(op.input(0));
+    auto value = loadFloat(dict["value"]);
```

Comment:
"value" is an optional param. I'd explicitly check if it's present and if not assign 0 to value.

Reply:
Won't `operator[]` do that for me?

Reply:
it will if it's an std::map, we have typedef for type of `dict`. It's kinda fragile.

Reply:
I'm still going to change it but just for everyone's info `dict` is a `std::unordered_map` and `operator[]` appears to have the same semantics: http://www.cplusplus.com/reference/unordered_map/unordered_map/operator[]/